add 219-colorization (#482)
openvinotoolkit/openvino_notebooks: notebooks/219-colorization/README.md

> # Colorization with OpenVINO

Reviewer: ```suggestion
# Photo Colorization with OpenVINO
```

![colorization](https://user-images.githubusercontent.com/30856589/159167043-5b78400b-c4ca-4a22-95c6-f67a3861ffe3.png)

> Colorization can turn a black and white image into a plausible color version of the image. Add color to black and white images, and bring the old images  back to life.

Reviewer: ```suggestion
Colorization turns a black and white image into a version with color. Add color to black and white images to bring old images back to life.
```
